packages/documentation/src/pages/forms/available-features-and-usage-guidelines.mdx

```diff
@@ -429,6 +429,7 @@ To configure this feature, place a `preSubmitInfo` object in the `formConfig`. T
 * `field`: The name of the form field for the required checkbox. This field has the value `true` in the submitted form data.
 * `label`: A text string or React element that labels the checkbox.
 * `error`: A text string or React element displayed as an error message if the user attempts to submit the form without checking the checkbox.
+* `CustomComponent`: A React element that will replace the default [`PrivacyAgreement`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms/components/PrivacyAgreement.jsx) inside the [SubmitController](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/platform/forms-system/src/js/review/SubmitController.jsx) component.
```

Comment:
Now it's the `PreSubmitSection` this overrides, not the `PrivacyAgreement`.

Reply:
Corrected